.gitignore

```diff
@@ -9,3 +9,4 @@ kafka-cookbook-*.tar.gz
 /**/vendor/**/bundle/
 /pkg
 TODO.md
+**/.DS_Store
```

Comment:
I'd say that this is not a relevant change, and I think it'd be better if you added that to your global `.gitignore`.